src/components/charts/shared/shapes/Circle.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Circle extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.padding = 0.5;
```

Comment:
Each padded value is different for each shape in order to keep their sizing, more or less, the same. This is only because each shape is created and centered slightly different than the other. This is the only way I could think to address this.

Reply:
I sorta suspect this is what's causing the symbols to be off center.  They don't look perfect in the following screen shots but they sure look closer.  Maybe I'm not thinking about this correctly? 

![Screen Shot 2021-05-24 at 5 45 58 PM](https://user-images.githubusercontent.com/8166060/119423406-0cd19c00-bcb8-11eb-86a0-5fb7198b2a8c.png)
![Screen Shot 2021-05-24 at 5 46 35 PM](https://user-images.githubusercontent.com/8166060/119423400-0b07d880-bcb8-11eb-883d-d083891d931a.png)
![Screen Shot 2021-05-24 at 5 46 17 PM](https://user-images.githubusercontent.com/8166060/119423402-0c390580-bcb8-11eb-9c16-9987a2fa7058.png)
![Screen Shot 2021-05-24 at 5 45 35 PM](https://user-images.githubusercontent.com/8166060/119423407-0cd19c00-bcb8-11eb-80f9-d19b8ec501c8.png)

Reply:
I worked on centering each shape yesterday and come up with a good solution. I haven't pushed up that work yet.